testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/plain/resource/DefaultNumberWriterResource.java

```diff
@@ -12,27 +12,27 @@
    @GET
    public Response get(@PathParam("type") String type) throws Exception {
       if ("Byte".equals(type)) {
-         return Response.ok().entity(new Byte((byte) 123)).build();
+         return Response.ok().entity(Byte.valueOf((byte) 123)).build();
```

Comment:
This comment is meant for all the primitives here, they don't need to be wrapped in the `*.valueOf()`. They can just be cast and they'll be autoboxed.